pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -239,6 +239,20 @@ type ClusterSpec struct {
 
 	// Tags for AWS instance groups
 	CloudLabels map[string]string `json:"cloudLabels,omitempty"`
+
+	// Hooks for custom actions e.g. on first installation
+	Hooks []HookSpec `json:"hooks,omitempty"`
```

Comment:
The name does not sound like system scripts, etc. Any other ideas? Run or system in the name?  RunHook, RunOnSystemHook?